src/library/scala/collection/immutable/TreeSet.scala

```diff
@@ -234,9 +275,20 @@ final class TreeSet[A] private[immutable] (private[immutable] val tree: RB.Tree[
     }
   }
 
-  private [collection] def removeAll(ts: TreeSet[A]): TreeSet[A] = {
-    assert (ordering == ts.ordering)
+  private [collection] def removeAll(xs : GenTraversableOnce[A]): TreeSet[A] = xs match {
+    case ts: TreeSet[A] if ordering == ts.ordering =>
     newSetOrSelf(RB.difference(tree, ts.tree))
```

Comment:
Indent this line

Reply:
done - I do with that we could just use the standard formatter, but the IJ configured formatter isn't very sensible IMO